micromamba/src/activate.cpp

```diff
@@ -28,23 +28,35 @@ set_activate_command(CLI::App* subcom)
         [&]()
         {
             std::string guessed_shell = guess_shell();
-            std::string shell_hook_command = "", shell_hook = "";
+            std::string shell_hook_command = "", shell_hook = "", shell_hook_dev_mode_cmd = "",
+                        shell_hook_dev_mode = "";
 
             if (guessed_shell == "powershell")
                 shell_hook_command
                     = "micromamba.exe shell hook -s powershell | Out-String | Invoke-Expression";
             else
+            {
                 shell_hook_command
                     = "eval \"$(micromamba shell hook --shell=" + guessed_shell + ")\"";
 
+                shell_hook_dev_mode_cmd = "eval \"$(./micromamba/micromamba shell hook --shell="
+                                          + guessed_shell + ")\"";
+
+                shell_hook_dev_mode = unindent((R"(
+
+                Or in dev mode, in your build directory:
+                    $ )" + shell_hook_dev_mode_cmd)
+                                                   .c_str());
+            }
+
             if (guessed_shell != "cmd.exe")
             {
                 shell_hook = unindent((R"(
 
                 To initialize the current )"
                                        + guessed_shell + R"( shell, run:
                     $ )" + shell_hook_command
-                                       + R"(
+                                       + shell_hook_dev_mode + R"(
```

Comment:
This is going to be shown to all (non-dev) users? If so I would suggest a better approach would be to put that information in the docs.

Reply:
> This is going to be shown to all (non-dev) users? 
Yes.

> If so I would suggest a better approach would be to put that information in the docs.
Well I thought about it but since the problem occurred while trying to activate an env with the built micromamba I had this message error (which wasn't solving the problem). So I think that having an error message with the right suggested command would really help because it's the first thing that I would try (follow the suggestion, before searching in the docs), but that doesn't mean not putting it there as well.